### ISSUE TITLE
Ubuntu Bionic rapidly approaching EOL

### ISSUE THREAD

**Issue by JWHolding** [mendix/docker-mendix-buildpack#153]
Hi, we used the ubuntu images as part of this buildpack. I note that Bionic is going to become EOL April next year.

Can we have an updated base image that uses ubuntu 20.04 or 22.04 to continue being in support?

**Comment by JWHolding** [comment]
should be noted that i've tried downloading and tweaking the dockerfile but libssl 1.0 is obsolete and I wasn't sure how much it was requried.

removing that when i run the tests from the make file i get the following error:

```
tests-mendixapp-1  | Traceback (most recent call last):
tests-mendixapp-1  |   File "/usr/lib/python3.10/pkgutil.py", line 417, in get_importer
tests-mendixapp-1  |     importer = sys.path_importer_cache[path_item]
tests-mendixapp-1  | KeyError: PosixPath('/opt/mendix/buildpack/buildpack/databroker/config_generator/scripts/generators/debezium_configs')
tests-mendixapp-1  |
tests-mendixapp-1  | During handling of the above exception, another exception occurred:
tests-mendixapp-1  |
tests-mendixapp-1  | Traceback (most recent call last):
tests-mendixapp-1  |   File "/opt/mendix/buildpack/buildpack/start.py", line 10, in <module>
tests-mendixapp-1  |     from buildpack import databroker, util
tests-mendixapp-1  |   File "/opt/mendix/buildpack/buildpack/databroker/__init__.py", line 14, in <module>
tests-mendixapp-1  |     from buildpack.databroker import business_events, connect, streams
tests-mendixapp-1  |   File "/opt/mendix/buildpack/buildpack/databroker/connect.py", line 17, in <module>
tests-mendixapp-1  |     from buildpack.databroker.config_generator.scripts.generators import (
tests-mendixapp-1  |   File "/opt/mendix/buildpack/buildpack/databroker/config_generator/scripts/generators/debezium.py", line 1, in <module>
tests-mendixapp-1  |     from buildpack.databroker.config_generator.scripts.generators.debezium_configs.debezium_interface import (
tests-mendixapp-1  |   File "/opt/mendix/buildpack/buildpack/databroker/config_generator/scripts/generators/debezium_configs/__init__.py", line 9, in <module>
tests-mendixapp-1  |     for (_, module_name, _) in iter_modules([package_dir]):
tests-mendixapp-1  |   File "/usr/lib/python3.10/pkgutil.py", line 129, in iter_modules
tests-mendixapp-1  |     for i in importers:
tests-mendixapp-1  |   File "/usr/lib/python3.10/pkgutil.py", line 421, in get_importer
tests-mendixapp-1  |     importer = path_hook(path_item)
tests-mendixapp-1  |   File "<frozen importlib._bootstrap_external>", line 1632, in path_hook_for_FileFinder
tests-mendixapp-1  |   File "<frozen importlib._bootstrap_external>", line 1504, in __init__
tests-mendixapp-1  |   File "<frozen importlib._bootstrap_external>", line 182, in _path_isabs
tests-mendixapp-1  | AttributeError: 'PosixPath' object has no attribute 'startswith'
tests-mendixapp-1 exited with code 0
```

**Comment by JWHolding** [comment]
This was resolved by #163
